Dockerfile

```diff
@@ -93,6 +90,13 @@ COPY my_init.d/prepare-aiidalab.sh /etc/my_init.d/80_prepare-aiidalab.sh
 COPY opt/start-notebook.sh /opt/
 COPY service/jupyter-notebook /etc/service/jupyter-notebook/run
 
+# Activate appmode
+WORKDIR /opt/
+RUN git clone https://github.com/oschuett/appmode.git
```

Comment:
Please check out a fixed version

Reply:
We need to wait until the version is released by Ole.

Reply:
you can just use the sha1 hash for the moment